R/pkg/R/DataFrame.R

```diff
@@ -2121,7 +2121,7 @@ setMethod("arrange",
           })
 
 #' @rdname arrange
-#' @name orderBy
```

Comment:
Oh I see what you meant. Well if @name is added different from @rdname, then \alias{orderBy} appears both in orderBy.Rd and arrange.Rd. I guess that's the root of the complaint.